libraries/AP_RangeFinder/AP_RangeFinder.h

```diff
@@ -193,9 +193,14 @@ class RangeFinder
         Good
     };
 
+    static constexpr int8_t SIGNAL_QUALITY_MIN = 0;
```

Comment:
this is not used.

Reply:
Correct at this point. But it might be used in the future and it helps document the valid range of signal_quality_pct values.

Reply:
Hi @ptrmu, it's a bit of a golden rule in AP though that we don't add code that's not used.  We can always add it later when it is necessary.

Reply:
Sure. Will remove it.

Reply:
A comment would be good for documenting the range tho ;)